framework/src/base/AuxiliarySystem.C

```diff
@@ -396,14 +405,17 @@ AuxiliarySystem::computeNodalVars(ExecFlagType type)
       solution().close();
       _sys.update();
     }
+    PARALLEL_CATCH;
+    Moose::perf_log.pop(compute_aux_tag, "Execution");
   }
-  PARALLEL_CATCH;
-  Moose::perf_log.pop("update_aux_vars_nodal()", "Execution");
 
-  // Boundary Nodal AuxKernels
-  Moose::perf_log.push("update_aux_vars_nodal_bcs()", "Execution");
-  PARALLEL_TRY {
-    if (nodal.hasActiveBoundaryObjects())
+  if (nodal.hasActiveBoundaryObjects())
```

Comment:
I've been looking at the warehouses and am convinced that the various `Warehouse::has...Objects()` methods are all symmetric in parallel runs. However, I have a similar check for a StoredRange elsewhere in this PR that I'm pretty sure isn't symmetric. I'd probably better undo that one...